user_delete_confirm.php

```diff
@@ -0,0 +1,32 @@
+<?php
+require "header.php";
+$user_id = $_POST['user_id'];
+$sql = 'SELECT * FROM users where id = :id';
```

Comment:
```suggestion
$sql = 'SELECT * FROM users where id = :id AND delete_flag = 0';
```
一応削除されてないかどうかのチェックも入れておきましょう